public/docs/_examples/tutorial/dart/pubspec.yaml

```diff
@@ -0,0 +1,15 @@
+name: angular2_tour_of_heroes
+version: 0.0.1
+environment:
+  sdk: '>=1.13.0 <2.0.0'
+dependencies:
+  angular2: 2.0.0-beta.1
+  browser: ^0.10.0
+  dart_to_js_script_rewriter: ^0.1.0+4
```

Comment:
This breaks Dartium though, right?

Having it as a one-off to show how to handle production is goodness. We shouldn't have it in every pubspec if it means Dartium won't work.